packages/client/src/client.js

```diff
@@ -271,6 +271,8 @@ export default class PostRPCClient {
 						}
 
 						call.reject(this.timeoutResponse(call.id)['error'])
+					} else {
+						throw new Error(`Unable to find or assign a handler for this call: ${JSON.stringify(call)}`)
```

Comment:
i think it should still be removed from the queue - else you get the same error thrown every 250 ms because of the timeout handler

Reply:
good call.

Reply:
I think I was just thinking that the program should crash with that error, but I guess there might still be some life in it even with that one call failing.